packages.json

```diff
@@ -22277,5 +22277,17 @@
     "description": "rodster",
     "license": "MIT",
     "web": "https://github.com/j-a-s-d/rodster"
+  },
+  {
+    "name": "xgboost.nim",
+    "url": "https://github.com/jackhftang/packages.git",
```

Comment:
Needs to be you package URL: https://github.com/jackhftang/xgboost.nim

Reply:
Ooops, silly copy & paste